requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
 scipy
 mne
-torch
\ No newline at end of file
+torch
+importlib-resources; python_version<"3.9"
```

Comment:
I don't think it makes much sense listing this here unless you pin it to a specific version, as it's listed as a package requirement in setup.cfg already

Reply:
If we want to keep `requirements.txt`, then it should have the same information as `setup.cfg`. The goal is to enable `pip install -r requirements.txt`. Now, personally, I don't like duplicating code/info across files.
So either we read the `requirements.txt` in `setup.py` and remove it from `setup.cfg` (as MNE) or we remove the `req` TXT files entirely.

I already know which option you would choose @hoechenberger ;)
@adam2392 Any preference?

Reply:
I always install things via `requirements.txt` file when doing dev work... so I feel like that's nice to have(?) 

@hoechenberger is your proposal to have both?

Reply:
But I suppose it's a longer discussion. Feel free to merge in either case ;)

Reply:
I see... I guess I'm in favor of `setup.cfg` and then extra requirements files for devs(?)

Reply:
btw this is what the official Python docs say:

https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#requirements-files

And this Medium post also gives an opinion:
https://towardsdatascience.com/requirements-vs-setuptools-python-ae3ee66e28af#bb0e

Reply:
So we still need the req files for doc and testing since those are useful but based on skimming these two we should get rid of requirements.txt file in favor of setup.cfg

Sounds good to me! I like simplicity

Reply:
No you could just move those to extras_require too

dev
docs

Something like that :)

But it's opinionated! So in the end it comes down to your personal preferences

Reply:
Personally, I also appreciate the `req.txt` files for dev. 
But I do think it would be nice if we can avoid duplicating the req. between `setup.cfg` and the `.txt` files. Removing the base `requirements.txt` in favor of `setup.cfg` while preserving the doc and test `.txt` works for me.

Reply:
Sounds like a good compromise for now!